python/pyspark/ml/tests/test_image.py

```diff
@@ -74,17 +74,14 @@ def setUpClass(cls):
         super(ImageFileFormatOnHiveContextTest, cls).setUpClass()
         cls.hive_available = True
         # Note that here we enable Hive's support.
-        cls.spark = None
         try:
-            cls.sc._jvm.org.apache.hadoop.hive.conf.HiveConf()
+            cls.spark = SparkSession.builder.enableHiveSupport().getOrCreate()
```

Comment:
@HyukjinKwon I may need your input on whether this is the right way to 'replace' use of HiveContext; is this test needed anymore even?

Reply:
@srowen, I think actually you can remove this whole class, `ImageFileFormatOnHiveContextTest`.
This test was added as of [SPARK-22651](https://jira.apache.org/jira/browse/SPARK-22651) to test `ImageSchema.readImages` but it was removed as of [SPARK-25382](https://jira.apache.org/jira/browse/SPARK-25382).

So, I think it's fine to remove this whole test class.